RafRate.Data/Entities/UserEntity.cs

```diff
@@ -5,5 +5,21 @@ namespace RafRate.Data.Entities
     public class UserEntity : IEntity
```

Comment:
Will you use identity?
If you will use, comments in this file is unnecessary, becase entity will replace with identity user